test/integration/targets/vmware_dvs_portgroup/tasks/main.yml

```diff
@@ -385,3 +385,136 @@
   assert:
     that:
         - dvs_pg_result_0017.changed
+
```

Comment:
Agreed with @mariolenz, you can just use a `when: vcsim is not defined`. You can either use a `block:` or, move all the tasks in a new playbook and do a:

```
- import_tasks: private_vlan.yaml
  when: vcsim is not defined
```

Reply:
Thanks for the tip! I used a block and also removed the extra vmware_dvs_portgroup_facts debug.